app/workers/active_payment_worker.rb

```diff
@@ -0,0 +1,12 @@
+class ActivePaymentWorker
+  include Sidekiq::Worker
+  def perform payment_id
+    @payment = Payment.find_by id: payment_id
+    if @payment
+      @payment.create_activation_digest
+      @payment.send_activation_email
+    else
+      Sidekiq.logger.error "logger.info : Cannot found payment}"
+    end
```

Comment:
```suggestion
    return Sidekiq.logger.error("logger.info : Cannot found payment}") unless @payment

    @payment.create_activation_digest
    @payment.send_activation_email
```
ngoài ra, đã ghi log error mà sao là `logger.info` nhỉ? thấy hơi sai sai, chị nghĩ là error luôn ấy chứ

Reply:
dạ vâng , em xóa logger.info rồi ạ